### ISSUE TITLE
Where is the search on DOC site?

### ISSUE THREAD

**Issue by ttodua** [savonet/liquidsoap#1433]
yeah, stupid question, however, couldnt find any search-box in DOC site: 
https://www.liquidsoap.info/doc-1.4.3/

**Comment by rrolla** [comment]
Search is on API reference: https://www.liquidsoap.info/doc-1.4.3/reference.html

**Comment by ttodua** [comment]
yes but... search box should be in top on any page of site ;.

**Comment by smimram** [comment]
Ok! Reopening.

**Comment by hayden-t** [comment]
yes please this driving me mad, very hard to use docs

**Comment by smimram** [comment]
I have just added a search box in the upper-right corner.

**Comment by ttodua** [comment]
thanks

**Comment by rrolla** [comment]
> I have just added a search box in the upper-right corner.

Is it expected that it uses google custom search?

![image](https://user-images.githubusercontent.com/3932546/146648036-1010cb14-275d-4d9f-a548-cc99639ecb45.png)

**Comment by ttodua** [comment]
btw, sometimes google has the best search engine, which even site cores can't have in some cases. But I can't say about the LS cms engine.

**Comment by smimram** [comment]
Yes, we have a static website, which has many advantages, but it makes things like searching more difficult. If you have better suggestions than google search we're open to those!
